.pre-commit-config.yaml

```
@@ -28,3 +28,12 @@ repos:
       #   entry: flake8
       #   types: [python]
       #   language: system
+  - repo: https://github.com/johann-petrak/licenseheaders.git
+    rev: 'v0.8.8'
+    hooks:
+      - id: licenseheaders
+        args: ["-t", ".github/templates/apache-2.tmpl", "-y2020-2023",
+        "-o", "Key4hep-Project",
+        "-n", "Key4hep",
+        "-u", "https://key4hep.github.io/key4hep-doc/",
+        "-x", ".github/*", ".pre-commit-config.yaml", "README.md", "doc/ReleaseNotes.md"]
```

```suggestion
        "-x", ".github/*", ".pre-commit-config.yaml", "README.md", "doc/ReleaseNotes.md", "-f"]
```
Make it run on just the file that pre-commit gives it.